views/beers.hbs

```diff
@@ -0,0 +1,16 @@
+
+<body>
```

Comment:
You don't need a <body> here as every template engine is going to be rendered as part of the 'layout.hbs' inside {{{body}}}